test/Microsoft.Azure.ServiceBus.UnitTests/SenderReceiverClientTestBase.cs

```diff
@@ -124,7 +124,7 @@ public abstract class SenderReceiverClientTestBase
             TestUtility.Log("Sleeping 10 seconds...");
             await Task.Delay(TimeSpan.FromSeconds(10));
 
-            DateTime lockedUntilUtcTime = await messageReceiver.RenewLockAsync(receivedMessages.First().LockToken);
+            var lockedUntilUtcTime = await messageReceiver.RenewLockAsync(receivedMessages.First().LockToken);
```

Comment:
I don't think `var` should ALWAYS be used. For me, I prefer `var` wherever the type is understood and obvious when you don't have prior knowledge of the whole system. `var size = 1` is obvious. But in this case unless you open this in a IDE, you cant be sure whether the object returned is DateTime / DateTimeOffset or any other custom implementation.